---
layout: default
---
# GB0230

## Graph-Break Type
*Short name describing what triggered the graph break*

Attempted to use torch.use_deterministic_algorithms(warn_only=True)

## Context
*Values or code snippet captured at the break point*

mode={mode}, warn_only={warn_only}

## Explanation
*Explanation of why the graph break was triggered*

Dynamo does not support this.

## Hints
*Hints on how to resolve the graph break*

- Remove param warn_only in function call torch.use_deterministic_algorithms.
- It may be possible to write Dynamo tracing rules for this code. Please report an issue to PyTorch if you encounter this graph break often and it is causing performance issues.


## Additional Information

<!-- ADDITIONAL INFORMATION START - Add custom information below this line -->

<!-- ADDITIONAL INFORMATION END -->

[Back to Registry](../index.html)
